ocs_ci/helpers/helpers.py

```diff
@@ -4770,9 +4770,7 @@ def is_rbd_default_storage_class(custom_sc=None):
     Returns:
         bool : True if RBD is set as the  Default storage class for the cluster, False otherwise.
     """
-    default_rbd_sc = (
-        constants.DEFAULT_STORAGECLASS_RBD if custom_sc is None else custom_sc
-    )
+    default_rbd_sc = constants.DEFAULT_STORAGECLASS_RBD if sc_name is None else sc_name
```

Comment:
@paraggit do we need this validation? because, in this case, this function is always called with `sc_name`. if condition to validate `sc_name` will always be not none until no sc is attached to PVC. Looks like always `default_rbd_sc` is always assigned with passed `sc_name`